utilities/post_migration.py

```
@@ -1401,6 +1472,20 @@ def check_vms(
+        # XFS compatibility check - verify XFSv4 crc=0 when xfs_compatibility enabled
+        if vm_ssh_connections and plan.get("xfs_compatibility"):
+            try:
+                check_vm_command_output(
+                    vm=vm,
+                    vm_ssh_connections=vm_ssh_connections,
+                    plan=plan,
+                    source_provider_data=source_provider_data,
+                    command=["/usr/sbin/xfs_info", "/sdb1"],
+                    expected_output="crc=0",
+                )
+            except Exception as exp:
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)